senza/templates/__init__.py

```diff
@@ -19,5 +23,7 @@ def get_templates() -> dict:
             # ignore bad entry points
             continue
         else:
-            template_modules[e.name] = module
+            # make sure the entry point resolves to a module with the essential interface functions
+            if isinstance(module, ModuleType) and has_functions(module, ('gather_user_variables', 'generate_definition')):
+                template_modules[e.name] = module
```

Comment:
Would be nice to log in case templates do not conform with expectations. It is good for debugging, etc.